### PR TITLE
Fix validation of joining race

### PR DESCRIPTION
**Pull request by edykim**
Hi Geokrety! I just found your project via hacktoberfest label. The website looks really fun and enjoyable. I hope that I can help some of the parts.

This PR is related #2 . I added validating code for the joining race page. Please review the changes.

### CODE REVIEW

**Comment by kumy** [comment]
@edykim thanks for taking time to contribute. We'll review your PR soon.

File: website/race_join.php [hunk 1]
```diff
@@ -106,19 +106,25 @@
...
+        if (!$row) {
+            $errors[] = _('Unknown race');
+            throw new Exception();
+        } elseif ($private == 1 and $kret_haslo == '' and ($raceOwner != $userid)) {
             $errors[] = _('This race is private. Please provide the password');
             throw new Exception();
         } elseif ($private == 1 and $kret_haslo != $haslo) {
             $errors[] = _('Wrong password');
             throw new Exception();
+        } elseif ($status !== 0) {
```
Comment (kumy) [inline website/race_join.php]: This test will not work.  When you select data from a MySQL database using PHP the datatype will always be converted to a string.

Comment (edykim) [reply website/race_join.php]: Yes, that was right. Thanks!

**Comment by kumy** [comment]
@boly38 agree to merge?

**Comment by kumy** [comment]
Merged. @edykim Thanks!

Will be deployed on production during the weekend. In the meantime, it'll be available on https://rec.geokrety.org

**Comment by edykim** [comment]
Awesome! Thank you all @kumy @boly38!

**Pull request merged**